embeds/client-examples/node/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:18
+FROM node:16
```

Comment:
There is an issue with node client and node 18

Reply:
We can update this whenever that node issue gets addressed